src/main/java/com/snowflake/kafka/connector/internal/InternalUtils.java

```diff
@@ -299,6 +299,9 @@ protected static Properties generateProxyParametersIfRequired(Map<String, String
         proxyProperties.put(SFSessionProperty.PROXY_USER.getPropertyKey(), username);
         proxyProperties.put(SFSessionProperty.PROXY_PASSWORD.getPropertyKey(), password);
       }
+      // There is a change in JDBC version 3.13.31 which causes NPE if this is not added.
```

Comment:
Do we have a JIRA created for the owner to fix this? We shouldn't have this assumption, this is a behavior change

Reply:
Will create a issue on JDBC. 
I think this is more of a breaking change than a behavior change. Since we have a workaround, its okay to add this here but I agree it should be fixed by JDBC.

Reply:
Opened a PR in JDBC: https://github.com/snowflakedb/snowflake-jdbc/pull/1559